extensions/amp-story-shopping/amp-story-shopping.md

```diff
@@ -145,6 +145,10 @@ Sets the color of the CTA button and drawer.
 <img alt="theme" src="https://user-images.githubusercontent.com/3860311/161297644-f25837e0-82ec-4960-a219-edfaf7d09fd9.png"
 layout="intrinsic" width="800" height="135">
 
+#### `cta-text` {string} optional
+
+Sets the CTA Text for the shopping attachment page outlink.
```

Comment:
```suggestion
Sets the Call To Action text for the shopping attachment page outlink.
```

Reply:
There are multiple CTAs in this file - I went ahead and changed all of them to Call To Action.